ioc_copier/ioc_copier.py

```diff
@@ -0,0 +1,104 @@
+from shutil import copytree
+from pathlib import Path
+import os
+import re
+ioc = "DFKPS"  # The name of the ioc to run, folders to copy will be of the form, <ioc>-IOC-01App and ioc<ioc>-IOC-01
+start_copy = 11  # The folder to copy (avoid IOC1 as other IOCs usually reference this rather than copying it directly)
```

Comment:
This fails for me when set to an integer lower than 10 02, 03, 04 etc as integers beginning with 0 won't be recognised as integers and if entering 6, a 0 is not placed at the start meaning for the EUROTHRM IOC, when trying to copy 06, the directory is not found.